OpenMage/magento-lts
# Bugfix: cant create order from admin backend

### Description (*)


### Related Pull Requests


1. See OpenMage/magento-lts#4266

### Fixed Issues (if relevant)

1. Fixes OpenMage/magento-lts#4347

### Manual testing scenarios (*)

1. create order from admin backend
2. ...

@Hanmac  please double-check. Maybe something is wrong at my side.
